app/containers/TracksContainer/index.js

```diff
@@ -1,5 +1,159 @@
-import React from 'react';
+import React, { memo, useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { injectIntl } from 'react-intl';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+import { injectSaga } from 'redux-injectors';
+import { createStructuredSelector } from 'reselect';
+import styled from 'styled-components';
+import { tracksContainerCreators } from './reducer';
+import tracksContainerSaga from './saga';
+import { selectTrackName, selectTracksContainer, selectTracksData } from './selectors';
+import { debounce, get, isEmpty } from 'lodash';
+import { Input, Card, Skeleton } from 'antd';
+import T from '@components/T';
+import For from '@app/components/For';
+import If from '@app/components/If';
+import TrackComponent from '@app/components/TrackComponent/index';
 
-export default function TracksContainer() {
-  return <div> TracksContainer page </div>;
+const { Search } = Input;
+
+const TitleCard = styled(Card)`
+  && {
+    margin: 10px 0;
+    max-width: ${props => props.maxwidth}px;
+    color: ${props => props.color}px;
```

Comment:
keep in px and why does the color have a px dimension ?

Reply:
Updated the changes